cmd/sops/main.go

```diff
@@ -1558,7 +1639,7 @@ func main() {
 			// While this check is also done below, the `err` in this scope shadows
 			// the `err` in the outer scope.  **Only** do this in case --decrypt,
 			// --rotate-, and --set are not specified, though, to keep old behavior.
-			if err != nil && !c.Bool("decrypt") && !c.Bool("rotate") && c.String("set") == "" {
+			if err != nil && !c.Bool("decrypt") && !c.Bool("rotate") && c.String("set") == "" && c.String("unset") == "" {
```

Comment:
Please remove this. `unset` is a new subcommand, and not a new flag. (We really don't want to add any more flags-that-should-be-subcommands.)